### ISSUE TITLE
Loki Decompose Finetuning

### ISSUE THREAD

**Issue by Tu4n-ph4m** [GuangyuWangLab2021/Loki#30]
Hi, thanks for your great work! I'm looking into finetuning the model. I understand you trained the model on open_clip but i'm not sure what train_csv format is what it entails. Unfortunately, I couldn't find the train_csv file in the data folder either. 

For example, if I wanted to create TNBC_txt_features_sc_finetune.csv. What would be the input for the training?
Thank you!

model_name='coca_ViT-L-14'
pretrained_weight_path='./data/basic_usage/checkpoint.pt'
train_csv = './data/loki_decompose/TNBC_data/checkpoint_tnbc/finetune_data.csv'
name = 'finetune_tnbc'

train_command = [
    'python', '-m', 'training.main',
    '--name', name,
    '--save-frequency', '5',
    '--zeroshot-frequency', '10',
    '--report-to', 'wandb',
    '--train-data', train_csv,
    '--csv-img-key', 'img_path',
    '--csv-caption-key', 'label',
    '--warmup', '10',
    '--batch-size', '64',
    '--lr', '5e-6',
    '--wd', '0.1',
    '--epochs', '5',
    '--workers', '16',
    '--model', model_name,
    '--csv-separator', ',',
    '--pretrained', pretrained_weight_path,
    '--lock-text-freeze-layer-norm',
    '--lock-image-freeze-bn-stats',
    '--coca-caption-loss-weight','0',
    '--coca-contrastive-loss-weight','1',
    '--val-frequency', '10',
    '--aug-cfg', 'color_jitter=(0.32, 0.32, 0.32, 0.08)', 'color_jitter_prob=0.5', 'gray_scale_prob=0'
]

**Comment by HXD3D0323** [comment]
I also have this question!

**Comment by osakemon** [comment]
Hi all,

Thank you for the question!

The finetune CSV has the same structure as the training CSV that each row is a paired gene sentence and image path.

Let us know if you have further questions.

Best,
Weiqing
